### ISSUE TITLE
Providing Linux packages (Flatpak, Snap or AppImage)

### ISSUE THREAD

**Issue by Luraktinus** [kometbomb/klystrack#234]
There are no general Linux packages available.

please package klystrack as Flatpak, Snap or AppImage

https://appimage.org
https://flatpak.org
https://snapcraft.io

doesn't really matter which format you choose, they all solve that problem

**Comment by kometbomb** [comment]
Here's the snap for klystrack: kometbomb/klystrack-snap

The resulting snap file is not available for download because it's quite
large.

2018-03-04 13:43 GMT+02:00 Luraktinus <notifications@github.com>:

**Comment by Luraktinus** [comment]
this is indeed nice... is it uploaded to the snapstore?

**Comment by Luraktinus** [comment]
also, kinda quick and unrelated question...
Why did you decide for Snap?
(i wanna find out why it finds so big adoption)

**Comment by kometbomb** [comment]
No, it's not currently uploaded anywhere (IIRC I had some trouble with that). I can't really remember the exact reasoning behind the choice but at least it was easy to set up and it's really easy to install the app.

**Comment by Luraktinus** [comment]
thanks...

how did you have troubles with it?

**Comment by alexmyczko** [comment]
Here's an AppImage http://sid.ethz.ch/appimage/Klystrack-x86_64.AppImage
